scripts/upload-file-to-s3.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+"""
+Usage:
+    scripts/upload-file-to-s3.py
+        <stage> <file_path> <bucket_category> [<remote_path>] [<remote_name>] [--dry-run] [--public]
+
+Options:
+    -h --help   Show this screen.
+    --stage=<stage>  The stage being updated
+    --file_path=<file_path>  Path of file to be saved in S3
+    --bucket_category=<bucket_category>  Bucket category, used to get name ie agreements/ communications/ documents
+    --remote_path=<remote_path>  Path to save the file at inside the bucket
```

Comment:
S3 seems to use the idea of objects having a key rather than there being a file hierarchy (http://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html). You could rename it here to reflect that. Although the concept of a file path is probably less confusing...